### PR TITLE
Added Authorization Code Grant tutorial

### PR DESCRIPTION
**Pull request by jhiner**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/api-auth/tutorials/authorization-code-grant.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+It is important to understand that the Authorization Code flow should only be used in cases such as a Regular Web Application where the Client Secret can be safely stored. In cases such as a Single Page Application, the Client Secret is available to the client (in the web browser), so the integrity of the Client Secret cannot be maintained. That is why the Implicit Grant flow is more appropriate in that case. 
+:::
+
+## Using the Access Token
+
+Once the `access_token` has been obtained it can be used to make calls to the Resource Server by passing it as a Bearer Token in the `Authorization` header of the HTTP request:
+
+``` js
+// Use the access token to make API calls
+var options = {
```
Comment (cabralmartin) [inline articles/api-auth/tutorials/authorization-code-grant.md]: Maybe we can replace this with a HAR so we get multi language support.

File: articles/api-auth/tutorials/authorization-code-grant.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+::: panel-warning A Note About Consent
+If your client is configured as a First Party Client and the "Allow Skipping User Consent" flag on the Resource Server (API) is checked in Dashboard, the user will not be prompted for consent. In this case, consent is technically being provided in the Dashboard by the tenant admin.
+:::
+
+## Exchanging the Authorization Code for an Access Token
+
+Now that you have an Authorization Code, you must exchange it for an Access Token that can be used to call your API. Using the Authorization Code (`code`) from the previous step, you will need to POST to the OAuth Token URL:
+
+```text
```
Comment (cabralmartin) [inline articles/api-auth/tutorials/authorization-code-grant.md]: We can replace this with a HAR

File: articles/api-auth/tutorials/authorization-code-grant.md (hunk 1 above)
Comment (ntotten) [reply articles/api-auth/tutorials/authorization-code-grant.md]: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#http-request-snippets

**Pull request merged**